core/src/main/scala/spire/optional/unicode.scala

```diff
@@ -5,8 +5,6 @@ import spire.algebra._
 import spire.algebra.lattice._
 import spire.math._
 
-import spire.macros.Ops
-
```

Comment:
Sure, let's do it.
This is not exercised in the tests and I considered dropping it, among other things because it isn't very clear what method is being called via the macro.
Ideally we'd do some tests and verify it behaves the same in scala 2 and 3

Reply:
I'm 100% okay to drop for Scala 3.